<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateDeleteFormation</class>
 <widget class="QDialog" name="UpdateDeleteFormation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>341</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="nomFormlabel">
        <property name="text">
         <string>Nom de la formation:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nomFormationLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="nbCreditTotalLabel">
        <property name="text">
         <string>Nombre de crédits total requis:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="CreditSspinBox">
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="nbCSlabel">
        <property name="text">
         <string>Nombre de crédits de  CS requis:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="CSspinBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="nbTMLabel">
        <property name="text">
         <string>Nombre de crédits de TM requis:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="TMspinBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nombre total de CS TM requis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="CSTMTotalSpinBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="nbTSHlebel">
        <property name="text">
         <string>Nombre de crédits de TSH requis:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="TSHspinBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="nbSPlabel">
        <property name="text">
         <string>Nombre de Crédis de SP requis:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="SPspinBox">
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="nbCrditsMaxLevel">
        <property name="text">
         <string>Nombre de crédits libre possible (MAX):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="creditsLibrEspinBox">
        <property name="maximum">
         <number>30</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QCheckBox" name="PCBcheckBox">
        <property name="text">
         <string>PCB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="PCBspinBox">
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QCheckBox" name="PSFcheckBox">
        <property name="text">
         <string>PSF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="PSFspinBox">
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="nbColTSHLabel">
        <property name="text">
         <string>Nombre colonnes différentes (TSH):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="nbColTSHspinBox">
        <property name="maximum">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="nbLLigneTSHLabel">
        <property name="text">
         <string>Nombre de ligne différente (TSH):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="nbLigneTSHspinBox">
        <property name="maximum">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QLabel" name="nbSemestreMaxPossibleLabel">
        <property name="text">
         <string>Nombre de semestre maximum autorisé:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="nbSemesMaXspinBox">
        <property name="maximum">
         <number>15</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="trueUpdate">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QLabel" name="msgAfterFormationDeletion">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>181</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="FormationcomboBox"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="upDateFormation">
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteFormation">
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
